app/common/gristUrls.ts

```diff
@@ -163,6 +163,13 @@ export interface OrgUrlInfo {
   orgInPath?: string;     // If /o/{orgInPath} should be used to access the requested org.
 }
 
+function isDocInternalUrl(host: string) {
+  if (!process.env.APP_DOC_INTERNAL_URL) { return false; }
+  const internalUrl = new URL('/', process.env.APP_DOC_INTERNAL_URL);
+  const internalHostnameAndMaybePort = internalUrl.hostname + (internalUrl.port ? `:${internalUrl.port}` : '');
```

Comment:
What is the difference between this and `internalUrl.host` (not `hostname`)?

Reply:
You're right, it works as well (and that's much simpler! :) ). Thanks for the hint, that's done